spinnaker/provider.go

```diff
@@ -1,19 +1,21 @@
 package spinnaker
 
 import (
+	"log"
+
+	spin_config "github.com/estebangarcia/spin/config"
```

Comment:
It might be better to call this spinconfig. Underscores in Go are pretty unusual.